src/file/source-map/extract-source-map.js

```diff
@@ -1,16 +1,30 @@
 var convert = require('convert-source-map');
 
 var sourceMapRegex = /\n\/(\*|\/)# sourceMappingURL=(.*)$/mg;
+var inlineBase64SourceMapRegex = /\n\/(\*|\/)# sourceMappingURL=data:application\/json(.*);base64,(.*)$/mg;
+
+function extract(code, mapFileDir) {
+	
+	var inlineBase64SourceMapMatch = code.match(inlineBase64SourceMapRegex);
+	
+	if(inlineBase64SourceMapMatch) {
+		
+        return {
+            code: code.replace(sourceMapRegex, ''),
+            map: convert.fromComment(inlineBase64SourceMapMatch[0]).toObject()
+        };
+		
+	}
 
-function extract(code) {
-
-    var sourceMap = code.match(sourceMapRegex);
+    var sourceMapMatch = code.match(sourceMapRegex);
 
-    if (sourceMap) {
+    if (sourceMapMatch) {
 
+		var sourceMap = convert.fromMapFileComment(sourceMapMatch[0], mapFileDir);

```

Comment:
sometimes this'll get `code` in with the base64 inlines maps - we'll look for those still, but if not then we now do this and get from file